### PR TITLE
BackupService: new option to delete Fuseki backups after copy

### PR DESCRIPTION
**Pull request by srosset81**
New `deleteFusekiBackupsAfterCopy` setting

### CODE REVIEW

File: src/middleware/packages/backup/index.js [hunk 1]
```diff
@@ -147,7 +164,7 @@ const BackupService = {
         }
       }
     },
-    /** Returns an array of file paths to the backups relative to `this.settings.localServer.fusekiBase`. */
+    // Returns an array of file paths to the backups relative to `this.settings.localServer.fusekiBase`.
```
Comment (Laurin-W) [inline src/middleware/packages/backup/index.js:167]: Out of curiosity, why the change from jsdoc to normal comments?

**Review by Laurin-W — APPROVED** [review]
Looks good! Looking forward to the stability improvements :)

File: src/middleware/packages/backup/index.js (hunk 1 above)
Comment (srosset81) [reply src/middleware/packages/backup/index.js:167]: Oh, sorry. I learnt that for, one-line comments, it's better to use `//` than `/* */`, as there is less risks of forgetting the closing `*/`, which can break other comments... What advantage is there to have jsdoc comments in that case ?
